markdown/events/2021/hackathon-march-2021.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Hackathon - March 2021
+subtitle: A virtual online hackathon to build nf-core together.
+type: hackathon
+start_date: "2021-03-22"
+start_time: "09:00 CET"
+start_date: "2021-03-22"
+end_time: "17:00 CET"
+location_name: Zoom, Slack & YouTube.
+location_url: '#how-it-works'
```

Comment:
?
That's a strange URL

Reply:
It's a soft-link to one of the section headers below